working/1426-extension-types/feature-specification-views.md

```diff
@@ -114,7 +121,7 @@ type `R` or some supertype of `R` (which is what we normally do). For
 example:
```

Comment:
There are a few places above here that also need updating, e.g. :
- line 29 "we call views" to "we call view classes"
- line 32 "declared by a new view decl." to "declared by a view class decl"
- line 33 "replacement or modification" to just "replacement"

Reply:
Done.